internal/xds/translator/tracing.go

```diff
@@ -41,6 +42,16 @@ func buildHCMTracing(tracing *ir.Tracing) (*hcm.HttpConnectionManager_Tracing, e
 	var providerConfig typConfigGen
 
 	switch tracing.Provider.Type {
+	case egv1a1.TracingProviderTypeDatadog:
+		providerName = envoyDatadog
+
+		providerConfig = func() (*anypb.Any, error) {
+			config := &tracecfg.DatadogConfig{
+				ServiceName:      tracing.ServiceName,
+				CollectorCluster: tracing.Destination.Name,
+			}
```

Comment:
is `CollectorHostname` also needed ?

Reply:
As I understand, `CollectorHostname` has default value which relates to `CollectorCluster`: https://github.com/envoyproxy/envoy/blob/b203e3e826af06e3639229f816f2a11245476728/api/envoy/config/trace/v3/datadog.proto#L40-L42.  Also I didn't find similar fields for `OpenTelemetry` and `Zipkin`

Reply:
ok, have you been able to manually test this, and does it work ?

Reply:
I see traces in Datadog APM from Envoy Proxy deployed with this gateway, and I found code which relates to this logic: https://github.com/envoyproxy/envoy/blob/39851cddac60be79f58ab33eafa35a1c9dda81a3/source/extensions/tracers/datadog/config.cc#L51-L58

Or I misunderstood the question and need somehow write a unit test for this